gke-nfs-lb/charts/v0.0.1/nfs-csi-lb/templates/csi-nfs-lb-driverinfo.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: storage.k8s.io/v1
+kind: CSIDriver
+metadata:
+  name: nfs.csi.k8s.io
```

Comment:
We should probably change the name and use gke.io suffix.

Reply:
done. 

what about the csi driver name? https://github.com/tyuchn/csi-driver-nfs/blob/master/pkg/nfs/nfs.go#L66 @msau42